examples/shapes_and_collections/arrow_guide.py

```diff
@@ -81,28 +83,41 @@
                                  mutation_scale=100,
                                  transform=axs[1].transAxes)
 axs[1].add_patch(arrow)
-axs[1].set_xlim(0, 2)
-axs[1].set_ylim(0, 2)
+axs[1].set(xlim=(0, 2), ylim=(0, 2))
 
 
 ###############################################################################
 # Head shape and anchor points fixed in data space
 # ------------------------------------------------
 #
-# In this case we use `.patches.Arrow`.
+# In this case we use `.patches.Arrow`, or `.patches.FancyArrow`.
 #
 # Note that when the axis limits are changed, the arrow shape and location
 # change.
+#
+# `.FancyArrow`'s API is relatively awkward, and requires in particular passing
+# ``length_includes_head=True`` so that the arrow *tip* is ``(dx, dy)`` away
+# from the arrow start.  It is only included in this reference because it is
+# the arrow class returned by `.Axes.arrow`.
 
 fig, axs = plt.subplots(nrows=2)
 
 arrow = mpatches.Arrow(x_tail, y_tail, dx, dy)
 axs[0].add_patch(arrow)
+arrow = mpatches.FancyArrow(
+    x_tail, y_tail - .4, dx, dy, width=.1, length_includes_head=True)
+axs[0].add_patch(arrow)
+axs[0].arrow(
+    x_tail + 1, y_tail - .4, dx, dy, width=.1, length_includes_head=True)
 
 arrow = mpatches.Arrow(x_tail, y_tail, dx, dy)
 axs[1].add_patch(arrow)
-axs[1].set_xlim(0, 2)
-axs[1].set_ylim(0, 2)
+arrow = mpatches.FancyArrow(
+    x_tail, y_tail - .4, dx, dy, width=.1, length_includes_head=True)
+axs[1].add_patch(arrow)
+axs[1].arrow(
+    x_tail + 1, y_tail - .4, dx, dy, width=.1, length_includes_head=True)
```

Comment:
I very much dig this example of showing how the patch is identical to the method.